virtcontainers/sandbox.go

```diff
@@ -17,8 +17,8 @@ import (
 	"syscall"
 
 	"github.com/containernetworking/plugins/pkg/ns"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
```

Comment:
I think this change is not needed, is this an error from a formatting tool?

Reply:
Yes. I will discard this.